ElasticaBundle/Command/OrchestraCreateSchemaCommand.php

```diff
@@ -32,10 +32,6 @@ protected function configure()
      */
     protected function execute(InputInterface $input, OutputInterface $output)
```

Comment:
I need this proxy method so I can launch it from the console command.

I don't think that you can bypass this